tests/tests.c

```diff
@@ -87,6 +87,8 @@ static const sir_cl_arg cl_args[] = {
 
 static sir_cl_config cl_cfg = {0};
 
+#include "tests_malloc_bsd.h"
```

Comment:
Plz move above to where other include statement is if possible

Reply:
These need to be above the main statement, because malloc config is already initialized once main is called.

Reply:
Right... I meant move it up to 36 w the other include

Reply:
Oh alright